internal/lagoondb/client.go

```diff
@@ -71,6 +71,7 @@ func (c *Client) EnvironmentByNamespaceName(ctx context.Context, name string) (*
 		project.name AS project_name
 	FROM environment JOIN project ON environment.project = project.id
 	WHERE environment.openshift_project_name = ?
+ 	AND environment.deleted = '0000-00-00 00:00:00'
```

Comment:
very minor nit: an extra space at the start of this line:
```suggestion
	AND environment.deleted = '0000-00-00 00:00:00'
```

Reply:
heh github commit message from ui fail
will fix